packages/core-flows/src/inventory/workflows/delete-inventory-items.ts

```diff
@@ -0,0 +1,13 @@
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { deAttachInventoryItemStep, deleteInventoryItemStep } from "../steps"
+
+export const deleteInventoryItemWorkflowId = "delete-inventory-item-workflow"
+export const deleteInventoryItemWorkflow = createWorkflow(
+  deleteInventoryItemWorkflowId,
+  (input: WorkflowData<string[]>): WorkflowData<string[]> => {
+    deleteInventoryItemStep(input)
```

Comment:
A question for maybe @carlos-r-l-rodrigues or @adrien2p: Wasn't the idea that using link modules will allow as to do cascade deletes? If so, is this workflow necessary, and how would it look like if not?

Reply:
I will sync with @pKorsholm tomorrow about it. After removing any linkable entity we also have to flag this to the Remote joiner delete any related link to that entity